source/manual/grafana.html.md

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Grafana
+section: Monitoring
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+Grafana lets us create nice dashboards using Graphite data.
+
+- Production
+  - [AWS](https://grafana.blue.production.govuk.digital)
+  - [Carrenza](https://grafana.publishing.service.gov.uk)
+- Staging
+  - [AWS](https://grafana.blue.staging.govuk.digital)
+  - [Carrenza](https://grafana.staging.publishing.service.gov.uk)
+- Integration
+  - [AWS](https://grafana.blue.integration.govuk.digital)
```

Comment:
```suggestion
  - [AWS](https://grafana.integration.publishing.service.gov.uk)
```

I think it's better to stick to this address since that'll become the AWS address for production and staging as well when we no longer have Carrenza environments.